api/config/config_prod.json

```diff
@@ -41,5 +41,8 @@
   "mandrill": {
     "fromEmail": "donotreply@researchallofus.org",
     "sendRetries": 3
+  },
+  "userRecentResourceConfig": {
+    "userEntrycount":10
```

Comment:
Do you think we'll ever update this or would need to update it out of band? If not, I might argue for just a constant. This config file has a lot of values that I'd argue should probably be constants.